statchatexpress/app.js

```diff
@@ -70,8 +88,35 @@ app.post("/users/login", async (req, res) => {
     if (user.Password !== Password) {
       return res.status(400).json({ message: "Incorrect password" });
     }
+    req.session.userId = user.dataValues.id;
+    req.session.save(function () {
+      console.log("userid:", req.session);
+      res.json({ message: "Login successful" });
+    });
+  } catch (err) {
+    res.status(500).json({ message: err.message });
+  }
+});
 
-    res.json({ message: "Login successful" });
+app.post("/users/profile", async (req, res) => {
+  try {
+    console.log("userprofileid:", req.session);
+    const { gamertag, platform } = req.body;
+    const userId = req.session.userId;
+    const user = await User.findByPk(userId);
+
+    req.session.reload(() => {
+      console.log("repopulated", req.session);
+    });
```

Comment:
I don't think we need this at all, please remove.

Reply:
facts